packages/zosfiles/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Zowe z/OS files SDK package will be documented in thi
 
 ## Recent Changes
 
+- Fixed an issue in the Copy.dataSetCrossLPAR() function where the `spacu` attribute of the copied data set was always set to `TRK`, regardless of the source data set's attributes. [#2412](https://github.com/zowe/zowe-cli/issues/2412)
```

Comment:
I'd put the function in backticks:  " ... in the `Copy.dataSetCrossLPAR()` function where ... "